packages/cli/test/unit/commands/deploy/index.test.ts

```diff
@@ -784,7 +783,7 @@ describe('deploy', () => {
       archive: undefined,
     };
 
-    it('--force', async () => {
+    it.todo('--force', async () => {
```

Comment:
<!-- Convention comments refer to a team, repo, or project convention that is not being followed. Mention the convention and how it has been violated.  -->
**convention (soft-blocking):** unskip?